### PR TITLE
Update maker UI to allow maker going long

### LINKED ISSUE #1516: Maker UI changes for allowing the taker to go short
The API already supports going long, but it's not enabled in the maker UI.

We need to:
- allow setting up both long and short offer prices instead of a single one,
- change the button name from 'create/update sell order' to 'create/update offer'

### PR DESCRIPTION
**Pull request by klochowicz**

### CODE REVIEW

**Review by rishflab — APPROVED** [review]

File: maker-frontend/src/MakerApp.tsx [hunk 1]
```diff
@@ -137,12 +154,13 @@ export default function App() {
...
                         <GridItem colSpan={2} textAlign="center">
                             <Button
-                                disabled={isCreatingNewCfdOrder || orderPrice === "0"}
+                                disabled={isCreatingNewCfdOrder || shortPrice === "0"}
                                 variant={"solid"}
                                 colorScheme={"blue"}
                                 onClick={() => {
                                     let payload: CfdSellOrderPayload = {
-                                        price: Number.parseFloat(orderPrice),
+                                        price: Number.parseFloat(shortPrice),
```
Comment (bonomat) [inline maker-frontend/src/MakerApp.tsx:174]: I think the field is called: `price_short`

File: maker-frontend/src/MakerApp.tsx [hunk 2]
```diff
@@ -49,12 +49,13 @@ export default function App() {
```
Comment (bonomat) [inline maker-frontend/src/MakerApp.tsx:51]: I'm confused about this commit. Is this logic maybe a bit outdated?

File: maker-frontend/src/MakerApp.tsx (hunk 1 above)
Comment (klochowicz) [reply maker-frontend/src/MakerApp.tsx:174]: not in the old endpoint! I'll be doing a follow-up on top of this that uses the new endpoint (same with the `funding_rate`...)

Comment (bonomat) [reply maker-frontend/src/MakerApp.tsx:174]: ah, got it. 👍

File: maker-frontend/src/MakerApp.tsx [hunk 3]
```diff
@@ -173,7 +173,7 @@ export default function App() {
                                     makeNewCfdSellOrder(payload);
                                 }}
                             >
-                                {order ? "Update Sell Order" : "Create Sell Order"}
+                                {order ? "Update Maker Offers" : "Create Maker Offers"}
```
Comment (bonomat) [inline maker-frontend/src/MakerApp.tsx]: Nit: we are in the maker frontend, so no need to mention this :) 

```suggestion
                                {order ? "Update Offers" : "Create Offers"}
```

Comment (klochowicz) [reply maker-frontend/src/MakerApp.tsx]: github has a bug and won't let me "commit suggestion" :( - I'll push a fixup!

File: maker-frontend/src/MakerApp.tsx (hunk 2 above)
Comment (klochowicz) [reply maker-frontend/src/MakerApp.tsx:51]: yeah, this was the starting point from @rishflab's PR. I have created 2 separate spread values in one of the further commits - it doesn't really matter much in the maker UI apart from actually seeing 2 different prices IMHO, as it's more of an example and dev testing env

File: maker-frontend/src/MakerApp.tsx [hunk 4]
```diff
@@ -165,6 +165,8 @@ export default function App() {
                                         max_quantity: Number.parseFloat(maxQuantity),
                                         // TODO: Populate funding rate from the UI
                                         funding_rate: (0.00002283 * 24), // annualized 20% by default to have some values
+                                        // TODO: Populate funding rate from the UI
+                                        daily_funding_rate_long: (-0.00002283 * 24), // annualized 20% by default to have some values
```
Comment (bonomat) [inline maker-frontend/src/MakerApp.tsx:181]: On purpose negative?

Comment (klochowicz) [reply maker-frontend/src/MakerApp.tsx:181]: yes, wanted to visualise the maker profiting both ways - happy to remove the negative sign if it's more misleading than helping!

File: maker-frontend/src/MakerApp.tsx [hunk 5]
```diff
@@ -105,20 +115,39 @@ export default function App() {
                             value={format(maxQuantity)}
                         />
 
-                        <Text>Offer Price:</Text>
+                        <Text>Maker Short Price:</Text>
```
Comment (bonomat) [inline maker-frontend/src/MakerApp.tsx:118]: I'm still super confused about the naming... 
Maker short price means, that's the price the maker offers to a taker who goes long?

**Review by bonomat — APPROVED** [review]
👍
I think we need to do something about the namings... It's not very intuitive...

File: maker-frontend/src/MakerApp.tsx (hunk 4 above)
Comment (bonomat) [reply maker-frontend/src/MakerApp.tsx:181]: no worries, this is just for testing anyways :)

File: maker-frontend/src/MakerApp.tsx (hunk 5 above)
Comment (klochowicz) [reply maker-frontend/src/MakerApp.tsx:118]: Yes. That's why I tried do be more elaborate - I could go even more verbose "Taker going long price" instead if that helps.

**Pull request merged**